apps/console/src/features/identity-providers/api/identity-provider.ts

```diff
@@ -47,7 +47,8 @@ import {
     MultiFactorAuthenticatorInterface,
     OutboundProvisioningConnectorInterface,
     OutboundProvisioningConnectorListItemInterface,
-    OutboundProvisioningConnectorMetaInterface
```

Comment:
There are lint issues in this file. Shall we fix them?

Reply:
Resolved lint issues